doc/releases/release-notes-3.3.rst

```diff
@@ -402,6 +408,7 @@ Drivers and Sensors
 * Counter
 
   * STM32 RTC based counter should now be configured using device tree.
+  * Added driver for GigaDevice GD32 SoCs.
```

Comment:
RTC driver also classified 'Counter driver'.
I think it should be better to explicitly note 'Timer'.